### ISSUE TITLE
Strange warnings in the log [MC version 1.19.2]

### ISSUE THREAD

**Issue by irish-brigid** [TheComputerizer/Music-Triggers#193]
Like the title says. I'm not sure what trigger could be looking at these methods for parameters or what effect that might have on what songs get played. It seems I get a lot of repeated songs even when setting universal "play-once" to 5.

> [21:47:10:162] [Music Triggers/WARN ]: Channel[example] - Parameter mods.thecomputerizer.theimpossiblelibrary.common.toml.Variable@1eb93150 is not accepted for trigger time so it will be skipped!
> [21:47:10:162] [Music Triggers/WARN ]: Channel[example] - Parameter mods.thecomputerizer.theimpossiblelibrary.common.toml.Variable@3db3f999 is not accepted for trigger time so it will be skipped!

**Comment by TheComputerizer** [comment]
Yeah there's a missing method for representing the Variable objects as strings for the log. It will be fixed in 7.0 along with there being much better logging in general.

**Comment by irish-brigid** [comment]
Ah, yeah, that'd do it.

Programming is *hard.*

**Comment by TheComputerizer** [comment]
Fixed in 7.0.0
